backend/lcfs/conftest.py

```diff
@@ -153,26 +154,24 @@ async def dbsession_factory(
 @pytest.fixture
 def fastapi_app(
     dbsession: AsyncSession,
-    fake_redis_pool: ConnectionPool,
+    fake_redis_client: aioredis.FakeRedis,
     set_mock_user,  # Fixture for setting up mock authentication
     user_roles: List[RoleEnum] = [RoleEnum.ADMINISTRATOR],  # Default role
 ) -> FastAPI:
     # Create the FastAPI application instance
     application = get_app()
     application.dependency_overrides[get_async_db_session] = lambda: dbsession
-    application.dependency_overrides[get_redis_pool] = lambda: fake_redis_pool
+    application.dependency_overrides[get_redis_client] = lambda: fake_redis_client
 
     # Set up application state for testing
-    application.state.redis_pool = fake_redis_pool
-    # application.state.db_session_factory = test_session_factory
+    application.state.redis_client = fake_redis_client
     application.state.settings = settings
 
     # Set up mock authentication backend with the specified roles
     set_mock_user(application, user_roles)
 
     # Initialize the cache with fake Redis backend
-    fake_redis = aioredis.FakeRedis(connection_pool=fake_redis_pool)
-    FastAPICache.init(RedisBackend(fake_redis), prefix="lcfs")
+    FastAPICache.init(RedisBackend(fake_redis_client), prefix="lcfs")
```

Comment:
Initialize without connection pool.

Reply:
the default param is a Redis instance not a connection pool.